src/DeviceInterfaceValues.tsx

```diff
@@ -394,6 +413,17 @@ const SendInterfaceDataModal = ({
                   <option value="true">true</option>
                   <option value="false">false</option>
                 </Form.Select>
+              ) : selectedMapping.type === 'datetime' ? (
+                <DatePicker
+                  selected={selectedDate}
+                  onChange={(date: Date | null) => handleValuesChange(date)}
+                  maxDate={maxDate}
+                  showTimeSelect
+                  timeFormat="HH:mm"
+                  dateFormat="yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
```

Comment:
The `dateFormat` is just for visualization. I would use `dateFormat="Pp"` as it's easier for users to read